geoportal/geoportailv3_geoportal/static-ngeo/js/offline/lux-offline/lux-offline-reload.component.ts

```diff
@@ -0,0 +1,66 @@
+import 'jquery';
+import 'bootstrap/js/modal.js';
+import i18next from 'i18next';
+import {LuxBaseElement} from '../../LuxBaseElement';
+import {html} from 'lit';
+import {customElement, state, query} from 'lit/decorators.js';
+import { LuxOfflineServiceInstance } from './lux-offline.service';
+
+@customElement('lux-offline-reload')
+export class LuxReloadAlert extends LuxBaseElement {
+
+  @state()
+  private status;
+
+  private prevStatus;
+
+  @query('.modal')
+  private modal: HTMLElement;
+
+  constructor() {
+    super();
+    //this.offlineService = new LuxOfflineService();
+    this.prevStatus = LuxOfflineServiceInstance.status$.getValue();
+    LuxOfflineServiceInstance.status$.subscribe((status)=> {
+      if (status != 'IN_PROGRESS' && this.prevStatus != undefined
+          && this.prevStatus != 'NO_INIT'
+        && status != this.prevStatus) {
+        $(this.modal).modal('show');
+      }
+      this.prevStatus = status;
+    });
+    $(this.modal).modal('hide');
+  }
+
+  render() {
+    return html`
+      <div class="modal" tabindex="-1" role="dialog">
+        <div class="modal-dialog offline-modal" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title">${i18next.t('App reload needed: Offline data updated successfully')}</h4>
```

Comment:
I think we still have to extract/translate the new strings (2nd step in https://github.com/Geoportail-Luxembourg/geoportailv3/tree/integration#translations).